prow/entrypoint/run.go

```diff
@@ -138,8 +140,21 @@ func (o Options) ExecuteProcess() (int, error) {
 		arguments = o.Args[1:]
 	}
 	command := exec.Command(executable, arguments...)
-	command.Stderr = output
-	command.Stdout = output
+
+	stdout, err := command.StdoutPipe()
+	if err != nil {
+		return InternalErrorCode, fmt.Errorf("could not create stdout pipe: %v", err)
+	}
+
+	stderr, err := command.StderrPipe()
+	if err != nil {
+		return InternalErrorCode, fmt.Errorf("could not create stderr pipe: %v", err)
+	}
+
+	multi := io.MultiReader(stdout, stderr)
+	scanner := bufio.NewScanner(multi)
+	dtLogger := log.New(output, "", 0)
```

Comment:
nit: Does using the logger add anything over writing to `output` directly with `fmt.Fprintf`?

Reply:
I went with `log` because I know it is [goroutine safe](https://golang.org/pkg/log/#Logger) and will preserve the log line order for concurrent writes. Since we are only running one goroutine, I *think* `fmt.Fprintf` should be fine. I prefer `log` because of these inherent benefits and because in this case we are performing logging operations.

Reply:
Instead of changing `command.Stderr = output` can we just decorate `processLogFile` to add the timestamps?

I'm less convinced we need/should add this to os.Stdout because stackdriver already adds timestamps for us.

Reply:
The *major* drawback to not including the timestamp in stdout is that it will not be available in the pod logs, so things like:
- The [`/log`](https://github.com/kubernetes/test-infra/blob/a1a5b03d5644aaf935fbb9aa833c94bfde013d30/prow/cmd/deck/main.go#L1240-L1274) endpoint (e.g. https://prow.k8s.io/log?job=pull-test-infra-verify-file-perms&id=1186766357568425984 )
- kubectl logs **some-pod**

will not have a timestamp. This reduces the benefit of the feature *slightly*.

Reply:
On second though, I thing this is configurable with the `--timestamps` flag, so this seems reasonable to not log timestamp to stdout.